src/components/InngestCommHandler.ts

```diff
@@ -434,9 +435,9 @@ export class InngestCommHandler<H extends Handler, TransformedRes> {
 
         const stepRes = await this.runStep(runRes.fnId, "step", runRes.data);
 
-        if (stepRes.status === 500) {
+        if (stepRes.status === 500 || stepRes.status === 400) {
```

Comment:
Should we support `4xx` e.g. `404` may be popular ? Maybe make this `stepRes.status >= 400 && stepRes.status <= 500`?

Reply:
This is abstracted from the user atm; they can't purposefully send any codes, so only `400` is possible in the `4xx` range atm, and any further info can be communicated via the error data passed.